codersdk/agentsdk/convert.go

```diff
@@ -348,7 +348,7 @@ func ProtoFromLog(log Log) (*proto.Log, error) {
 	}
 	return &proto.Log{
 		CreatedAt: timestamppb.New(log.CreatedAt),
-		Output:    log.Output,
+		Output:    strings.ToValidUTF8(log.Output, ""),
```

Comment:
I think we should probably replace invalid utf-8 with something, so it's obvious data is being deleted.

Reply:
Should we get cute and replace with something like ❌ or 😞 or 🤬, or just something like `<<invalid UTF-8>>`?